scripts/inputs/create_test_batch.py

```diff
@@ -38,17 +54,47 @@ def replace_output_dir(value, execution_bucket, outputs_dir):
     return value.replace(execution_bucket, outputs_dir)
 
 
+def split_bucket_subdir(directory):
+    # Parse -b URI input into top-level bucket name (no gs://) and subdirectory path
+    uri = urlparse(directory)
+    return uri.netloc, uri.path.lstrip("/")
+
+
+def create_file_list(list_key, values_list, file_list_bucket):
+    # create tmp file with paths list
+    list_suffix = ".txt"
+    fd, fname = tempfile.mkstemp(suffix=list_suffix)
+    with os.fdopen(fd, 'w') as listfile:
+        listfile.write("\n".join(values_list))
+
+    # upload file to GCS
+    client = storage.Client()
+    dest_bucket, subdir = split_bucket_subdir(file_list_bucket)
+    bucket = client.get_bucket(dest_bucket)
+    gcs_blob_name = os.path.join(subdir, list_key + list_suffix)
+    blob = bucket.blob(gcs_blob_name)
+    blob.upload_from_filename(fname)
+
+    # delete tmp file
+    os.unlink(fname)
+
+    # return the GCS path of the paths list file
+    return os.path.join("gs://", dest_bucket, gcs_blob_name)
+
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("metadata", help="GATKSVPipelineBatch metadata JSON file")
     parser.add_argument("--final-workflow-outputs-dir", help="If used, final_workflow_outputs_dir "
                                                              "option from Cromwell config file")
     parser.add_argument("--execution-bucket", help="Cromwell execution bucket, required if "
                                                    "using --final-workflow-outputs-dir")
+    parser.add_argument("--file-list-bucket", help="Bucket to which to upload file lists")
```

Comment:
What do you think about setting a default for this to `--final-workflow-outputs-dir`?

Reply:
I thought about this but when I ran it I used `gs://gatk-sv-ref-panel-1kg/outputs/GATKSVPipelineBatch/38c65ca4-2a07-4805-86b6-214696075fef/` as opposed to `gs://gatk-sv-ref-panel-1kg/outputs/` because I wanted it to be tied to the workflow that produced the outputs, and I didn't want it to be overwritten next time (so that old versions are still valid). I could extract the workflow subdirectory from the files, but wasn't sure that was worth the effort